lib/routes/tongji/gs.ts

```diff
@@ -0,0 +1,66 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/gs',
+    categories: ['university'],
+    example: '/tongji/gs',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['gs.tongji.edu.cn/tzgg.htm', 'gs.tongji.edu.cn/'],
+        },
+    ],
+    name: '研究生院通知公告',
+    maintainers: ['sitdownkevin'],
+    handler,
+    url: 'gs.tongji.edu.cn/tzgg.htm',
+};
+
+async function getNoticeContent(item) {
+    const response = await got(item.link);
+    const $ = load(response.body);
+    const content = $('#vsb_content').text();
```

Comment:
```suggestion
    const content = $('#vsb_content').html();
```
Provide the HTML content instead of pure text.